{"name": "Негативная проверка доверенного номера не найденного клиента", "status": "passed", "description": "Проверка неудачного поиска клиента по доверенному номеру в формате 10 символов", "steps": [{"name": "Проверка сообщения об ошибке и кода ответа", "status": "passed", "start": 1718187369038, "stop": 1718187369038}], "parameters": [{"name": "proofNum", "value": "'0553565311'"}], "start": 1718187368928, "stop": 1718187369039, "uuid": "d645232e-4a7a-4b30-8302-8f7d69f1a2be", "historyId": "c36b2762d5fbe2fecf895e31e1b0d87d", "testCaseId": "1a34d257530ec8961396b7a17e6623ea", "fullName": "207_service_test.TestProofNumSuite#test_get_info_client_10_letter_fiz_in_proofNum_get_success_response", "labels": [{"name": "suite", "value": "(207 сервис) Получение информации по клиенту"}, {"name": "subSuite", "value": "Проверки по доверенному номеру"}, {"name": "host", "value": "PC-411-140322D"}, {"name": "thread", "value": "9896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "207_service_test"}]}